static/include/pyArborX_Point.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef PYARBORX_POINT_HPP
+#define PYARBORX_POINT_HPP
+
+#include <ArborX_Point.hpp>
+
+#include <pybind11/pybind11.h>
+#include <string.h>
+
+namespace py = pybind11;
+
+namespace pyArborXHelper
+{
+std::string pyPrintPoint(ArborX::Point const &p);
+}
+
+void generatePointWrapper(py::module &m);
```

Comment:
Did you mean to have that guy in the global namespace?

Reply:
I put the wrappers inside a `pyArborX` namespace in [53d8ee](https://github.com/arborx/pyArborX/pull/1/commits/53d8eeb022094b20e22781a7227d61371127ed58)